crates/subspace-service/src/sync_from_dsn/import_blocks.rs

```diff
@@ -136,7 +136,7 @@ where
             .get_segment_header(segment_index)
             .expect("Statically guaranteed to exist, see checks above; qed");
 
-        trace!(
+        debug!(
```

Comment:
This is a trace message to me. We already have a debug message about segment index being processed.